File: app/src/main/java/com/mehmetakiftutuncu/muezzin/models/WidgetType.java
package com.mehmetakiftutuncu.muezzin.models;

import com.mehmetakiftutuncu.muezzin.R;

/**
 * Created by akif on 08/07/16.
 */
public enum WidgetType {
    PRAYER_TIMES_HORIZONTAL(R.layout.widget_prayertimes_horizontal),
    PRAYER_TIMES_VERTICAL(R.layout.widget_prayertimes_vertical),
    PRAYER_TIMES_BIG(R.layout.widget_prayertimes_big);

    public final int layoutId;

    WidgetType(int layoutId) {
        this.layoutId = layoutId;
    }
}

File: app/src/main/java/com/mehmetakiftutuncu/muezzin/models/WidgetType.kt
package com.mehmetakiftutuncu.muezzin.models

import com.mehmetakiftutuncu.muezzin.R

enum class WidgetType(val layoutId: Int) {
    Horizontal(R.layout.widget_prayertimes_horizontal),
    Vertical(R.layout.widget_prayertimes_vertical),
    Big(R.layout.widget_prayertimes_big)
}